### PR TITLE
Stress test added

### PR DESCRIPTION
**Pull request by ricardocastrovidal**

### CODE REVIEW

File: DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java [hunk 1]
```diff
@@ -246,6 +258,33 @@ public void testSetKeywordWrongType() throws ClientProtocolException,
...
+                task);
+        ExecutorService executorService = Executors
+                .newFixedThreadPool(numberOfThreads);
+
+        List<Future<String>> futures = executorService.invokeAll(tasks);
+        List<String> resultList = new ArrayList<String>(futures.size());
+
+        for (Future<String> future : futures) {
+            // Throws an exception if an exception was thrown by the task.
+            resultList.add(future.get());
```
Comment (cquiroz) [inline DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java:283]: `resultList` doesn't seem to be used, shouldn't you check the contents?

**Comment by cquiroz** [comment]
There is a merge commit on the PR  8a24442, why's that?

**Comment by jluhrs** [comment]
I think that comes because we pulled some changes from my copy in Github to Ricardo's working copy. The same changes were already included in my original PR. I don't know how to fix it, but it shouldn't affect the final state of the code.

**Comment by cquiroz** [comment]
On the OCS we are using a different model for PRs where they are developed on separate branches. Normally you'd expect a PR going from `ricardocastrovidal:feature/xxx` to `gemini-hlsw:develop` rather than this case from `ricardocastrovidal:develop`. Before making the PR you may/should rebase the PR on `develop` to avoid those merge commits
The model is on the wiki at https://github.com/gemini-hlsw/ocs/wiki/OcsSourceCodeManagement

**Comment by swalker2m** [comment]
@cquiroz I agree, let's stick to the standard procedure unless and until there's a need to change it.  There should generally also be some JIRA issue to tie this work to and it should appear in the feature branch and in the title of the PR.

File: DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java (hunk 1 above)
Comment (swalker2m) [inline DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java:283]: We might as well use the `get()` variant with a timeout here.  I don't know how long is reasonable to wait but presumably this should be quick if all is well.

Can this test be run automatically though?  Does it require first setting up a server to connect to or is that handled somehow?

File: DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java [hunk 2]
```diff
@@ -246,6 +258,33 @@ public void testSetKeywordWrongType() throws ClientProtocolException,
...
+            }
+        };
+
+        List<Callable<String>> tasks = Collections.nCopies(numberOfThreads,
+                task);
+        ExecutorService executorService = Executors
+                .newFixedThreadPool(numberOfThreads);
+
+        List<Future<String>> futures = executorService.invokeAll(tasks);
+        List<String> resultList = new ArrayList<String>(futures.size());
```
Comment (swalker2m) [inline DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java:279]: Let's go ahead and mark everything `final` that can be final.  Also the explicit repeated type is not necessary as of Java 7.

``` java
final List<String> resultList = new ArrayList<>(futures.size());
```

**Comment by jluhrs** [comment]
The first PR done directly from develop because it was the first PR that filled the repository. But I agree that we should have used a feature branch for this one. All this mess was actually I trying to follow the standard procedure, but the rebase to develop didn't work as I expected. I guess I still don't get git.
I'll close the PR and try again.

**Pull request closed without merging**